config/optional/core.entity_form_display.message.private_message.default.yml

```diff
@@ -7,8 +6,6 @@ dependencies:
     - field.field.message.private_message.field_message_private_subject
     - field.field.message.private_message.field_message_private_to_user
     - message.template.private_message
-_core:
-  default_config_hash: 0IDVLOZjSksUi025JzU3KA6363ZSA3OK8jTdtFtK_x8
```

Comment:
Thanks for the PR. I see it says remove the hash in the docs now, cheers. Where does it say remove core `default_config_hash ` though?